aspnetcore/blazor/hybrid/routing.md

```diff
@@ -109,3 +109,118 @@ blazorWebView.UrlLoading +=
 ```
 
 :::zone-end
+
+:::zone pivot="maui"
+
+## Navigation among pages and Razor components
+
+This section explains how to navigate among .NET MAUI content pages and Razor components.
```

Comment:
I was about to comment that this would work for navigating between all .NET MAUI page types and Razor components. But the modification of the `App` class wouldn't be necessary for other page types. So you can ignore this comment :)